README.md

```diff
@@ -259,7 +259,20 @@ This will output styles for a lemon coloured button that has slate text, with a
 #### Icons
 If you're using bower, you can create an icon button for **any** icon in [fticons](https://registry.origami.ft.com/components/fticons/).
 
-If you're using the Build Service, currently supported icons are defined in the `$o-buttons-icons` variable in `scss/_variables.scss`. Limiting the concrete classes keeps the compiled CSS bundle small, but if you need an icon button that we don't currently support then please open an issue.
+If you're using the Build Service a limited number of button icons are available. Limiting the concrete classes keeps the compiled CSS bundle smaller, but if you need an icon button that we don't currently support then please open an issue. Build service icons currently include:
+- arrow-left
+- arrow-right
+- upload
+- tick
+- plus
+- warning
+- arrow-down
+- arrow-up
+- grid
+- list
+- edit
+- download
+- search
```

Comment:
it's nice to have these here. maybe we should have a comment in `variables.scss` part on icons to say "if you're changing this list you'll need to update the readme"